### ISSUE TITLE
Batch size 1 and value of Flattening Criterion

### ISSUE THREAD

**Issue by gitlabspy** [fqnchina/ImageSmoothing#14]
Great work! I have few questions, can you explain a bit for me please? 
1. Why batch size set to 1?
2. I try implement it in pytorch and found that the value of Flattening Criterion is pretty big, is that normal?

**Comment by RewindL** [comment]
1. I don't understand either.
2. Maybe you don't use mean() instead of sum() when you calculate the final loss (Flattening Criterion), which in paper loss divided by total pixel number N. During my pytorch training, Flattening Criterion is converged around 1e-3~1e-2，but I don't get pleasant results yet and still adjust paramaters.
